engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -92,7 +91,7 @@ class FlinkKafkaSource[T](
       flinkNodeContext: FlinkCustomNodeContext
   ): SourceFunction[T] = {
     new FlinkKafkaConsumerHandlingExceptions[T](
-      topics.asJava,
+      topics.map(_.name).toList.asJava,
```

Comment:
Yeah, I noticed we use `java.util.List` here and it's probably done on purpose, so I thought it's better to leave the String here instead of using `TopicName.ForSource`